### PR TITLE
Add requirements to readme.

### PR DESCRIPTION
**Pull request by tinproject**
Test PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,9 @@
 
 Autoconstruccion project at H4ckademy
 
-by Creepy Coconuts
\ No newline at end of file
+by Creepy Coconuts
```
Comment (tinproject) [inline README.md]: Awsome!!

**Comment by tinproject** [comment]
Looks good to me ;D

File: README.md (hunk 1 above)
Comment (AlvaroCaste) [inline README.md]: it's a f*ckin merge!

**Comment by AlvaroCaste** [comment]
LGTM

**Pull request merged**